tests/datasource/fluent/test_pandas_s3_datasource.py

```diff
@@ -136,12 +132,12 @@ def bad_regex_config(csv_asset: CSVAsset) -> tuple[re.Pattern, str]:
     return regex, test_connection_error_message
 
 
-@pytest.mark.filesystem
+@pytest.mark.aws_creds
 def test_construct_pandas_s3_datasource(pandas_s3_datasource: PandasS3Datasource):
     assert pandas_s3_datasource.name == "pandas_s3_datasource"
 
 
-@pytest.mark.filesystem
+@pytest.mark.aws_creds
```

Comment:
I think this and a lot of these other tests are using mocks and don't require `aws_creds`